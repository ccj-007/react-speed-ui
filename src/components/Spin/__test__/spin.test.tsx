import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Spin } from '../../index';

describe('test Spin component', () => {
  it('should render', () => {
    const { asFragment } = render(<Spin />);
    expect(asFragment()).toMatchSnapshot();
  });
});
