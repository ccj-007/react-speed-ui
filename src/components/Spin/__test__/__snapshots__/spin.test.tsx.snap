// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Spin component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-spin"
  >
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-spinner fa-1x speed-icon fa-spin"
      color="blue"
      data-icon="spinner"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M304 48c0-26.5-21.5-48-48-48s-48 21.5-48 48s21.5 48 48 48s48-21.5 48-48zm0 416c0-26.5-21.5-48-48-48s-48 21.5-48 48s21.5 48 48 48s48-21.5 48-48zM48 304c26.5 0 48-21.5 48-48s-21.5-48-48-48s-48 21.5-48 48s21.5 48 48 48zm464-48c0-26.5-21.5-48-48-48s-48 21.5-48 48s21.5 48 48 48s48-21.5 48-48zM142.9 437c18.7-18.7 18.7-49.1 0-67.9s-49.1-18.7-67.9 0s-18.7 49.1 0 67.9s49.1 18.7 67.9 0zm0-294.2c18.7-18.7 18.7-49.1 0-67.9S93.7 56.2 75 75s-18.7 49.1 0 67.9s49.1 18.7 67.9 0zM369.1 437c18.7 18.7 49.1 18.7 67.9 0s18.7-49.1 0-67.9s-49.1-18.7-67.9 0s-18.7 49.1 0 67.9z"
        fill="currentColor"
      />
    </svg>
    <div />
  </div>
</DocumentFragment>
`;
